src/ucs/config/parser.c

```diff
@@ -379,6 +398,111 @@ int ucs_config_sprintf_time(char *buf, size_t max, void *src, const void *arg)
     return 1;
 }
 
+int ucs_config_sscanf_bw(const char *buf, void *dest, const void *arg)
+{
+    double *dst     = (double*)dest;
+    char    str[16] = {0};
+    int     offset  = 0;
+    size_t  divider;
+    double  units;
+    double  value;
+    int     num_fields;
+
+    num_fields = sscanf(buf, "%lf%16s", &value, str);
+    if (num_fields < 2) {
+        return 0; /* bytes or bits? */
+    }
+
+    ucs_assert(num_fields == 2);
+
+    units = ucs_config_parser_quantity_prefix_value(str[0]);
+
+    if (!units) {
```

Comment:
it's bad practice to compare float with equal. looks like ucs_config_parser_quantity_prefix_value() should return the size_t

Reply:
fixed - used size_t